lib/engineering/base.rb

```diff
@@ -0,0 +1,4 @@
+module Engineering
```

Comment:
what's the purpose of this namespace? It looks to me like the name might be too open for many things

Reply:
Yeah, I was thinking of using it for all engineering maths that could not be in Math ruby's namespace

Reply:
ok, let's see how it matures